tools/create/scaffold/scaffold.go

```diff
@@ -0,0 +1,76 @@
+package scaffold
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func RunE(cmd *cobra.Command, args []string) error {
+	fmt.Fprintln(cmd.OutOrStdout(), "Validating")
+
+	if err := ValidateFlags(); err != nil {
+		return fmt.Errorf("%w\n", err)
+	}
+
+	if moduleConfigExists, err := ModuleConfigFileExists(); err != nil {
+		return fmt.Errorf("%w\n", err)
+	} else if moduleConfigExists && !Overwrite {
+		return fmt.Errorf("%w\n", errModuleConfigExists)
+	}
+
+	if manifestExists, err := ManifestFileExists(); err != nil {
+		return fmt.Errorf("%w\n", err)
+	} else if manifestExists {
+		fmt.Fprintf(cmd.OutOrStdout(), "The Manifest file already exists, reusing: %s\n", ManifestFilePath())
+	} else {
+		fmt.Fprintln(cmd.OutOrStdout(), "Generating Manifest file")
+		if err := GenerateManifest(); err != nil {
```

Comment:
There is no business layer interface here.
How to test this condition without actually invoking manifest generation? The purpose of this task was to come out with a unit-testable design, this one is not.